README.md

```diff
@@ -1,18 +1,26 @@
 # BigImageViewer
 
-[ ![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg) ](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
+[![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg)](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
```

Comment:
What's the point of those changes? And more changes after what I've created screenshots.

<img width="1376" alt="screen shot 2017-06-23 at 15 29 21" src="https://user-images.githubusercontent.com/3098704/27471006-d112c22c-5828-11e7-8baf-4494ead26621.png">
![Uploading Screen Shot 2017-06-23 at 15.29.14.png…]()
<img width="1384" alt="screen shot 2017-06-23 at 15 29 07" src="https://user-images.githubusercontent.com/3098704/27471005-d111b38c-5828-11e7-8592-0ed049ff8054.png">
<img width="1365" alt="screen shot 2017-06-23 at 15 28 57" src="https://user-images.githubusercontent.com/3098704/27471008-d112ff80-5828-11e7-9fa9-f0e17ede91dc.png">